frontend/src/pages/create-study-page/components/subject/Subject.tsx

```diff
@@ -26,22 +31,21 @@ const Subject: React.FC<SubjectProps> = ({ originalSubjects }) => {
     if (data?.tags) {
       const { tags } = data;
       const subjects = tags.filter(({ category }) => category.name === SUBJECT);
-      const etcTagId = subjects.find(tag => tag.name === 'Etc');
-
-      return (
-        <Select
-          id={SUBJECT}
-          defaultValue={(originalSubjects && originalSubjects[0].id) || etcTagId?.id}
-          fluid
-          {...register(SUBJECT)}
-        >
-          {subjects.map(({ id, description }) => (
-            <option key={id} value={id}>
-              {description}
-            </option>
-          ))}
-        </Select>
-      );
+      const etcTag = subjects.find(tag => tag.name === 'Etc');
+      if (!etcTag) {
+        throw new Error('기타 태그의 name이 변경되었습니다');
+      }
```

Comment:
```suggestion
      const etcTag = subjects.find(tag => tag.name === 'Etc') ?? '%ERROR%';
```

콘솔 등등으로 조금 더 약하게 에러를 표시하는 건 어떨까요?

Reply:
throw를 하면 app이 멈춰버리니까..!

Reply:
그런데 저희가 콘솔을 자주 보는건 아니라서, 조금 불안하네요.  
alert는 어떨까요?

Reply:
좋은 생각이네요 :D  

[e0904de](https://github.com/woowacourse-teams/2022-moamoa/pull/358/commits/e0904deca602d97a3643acb6b7d20aa8e99ae00c)